<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-3KPCD8VB" failures="3" tests="4" name="Test" time="42.319" errors="0" timestamp="2024-07-16T11:34:12 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="2.008"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.068">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.87">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.136">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
